frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -79,8 +79,8 @@ export const triggerPipeline = (
 export const reRunPipelineRun: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => ({
   label: 'Rerun',
   callback: () => {
-    const namespace = _.get(pipelineRun, 'spec.metadata.namespace');
-    const pipelineRef = _.get(pipelineRun, 'spec.spec.pipelineRef.name');
```

Comment:
spec.spec? how did this get in there.. :smile:

Reply:
Merge conflict maybe? 🤔

Reply:
yeah, probably